js/fc.js

```diff
@@ -37,6 +37,13 @@ var FAILSAFE_CONFIG;
 var RXFAIL_CONFIG;
 
 var FC = {
+    isRatesInDps: function () {
+        if (typeof CONFIG != "undefined" && CONFIG.flightControllerIdentifier == "INAV" && semver.gt(CONFIG.flightControllerVersion, "1.1.0")) {
```

Comment:
No need to check for INAV really, Configurator won't connect to anything but INAV so we can safely remove most of compatibility stuff from Cleanflight.